.circleci/run-stackrox-ci/10-run-stackrox-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+BRANCH=$1
+BUILD_NUM=$2
+
+comment_on_pr_with_ci_link() {
+    local ci_url=$1
+
+    wget --quiet https://github.com/joshdk/hub-comment/releases/download/0.1.0-rc6/hub-comment_linux_amd64
+    sudo install hub-comment_linux_amd64 /usr/bin/hub-comment
```

Comment:
[nit] Since hub comment is used in 2 separate jobs, we might want to move these two lines to a circleci command like this one:
```yaml
  install-hub-comment:
    steps:
    - run:
        name: Install HUB comment
        command: |
          wget --quiet https://github.com/joshdk/hub-comment/releases/download/0.1.0-rc6/hub-comment_linux_amd64
          sudo install hub-comment_linux_amd64 /usr/bin/hub-comment
```

That way we can later just add `- install-hub-comment` in any job that requires it and focus on using it rather than installing it.